testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeChangeDataFeedCompatibility.java

```diff
@@ -88,6 +91,40 @@ public void testUpdateTableWithCdf()
         }
     }
 
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_OSS, DELTA_LAKE_EXCLUDE_73, PROFILE_SPECIFIC_TESTS})
+    @Flaky(issue = DATABRICKS_COMMUNICATION_FAILURE_ISSUE, match = DATABRICKS_COMMUNICATION_FAILURE_MATCH)
+    public void testUpdateCdfTableWithNonLowercaseColumn()
+    {
+        String tableName = "test_updates_cdf_with_non_lowercase_" + randomNameSuffix();
+
+        onDelta().executeQuery("CREATE TABLE default." + tableName +
+                "(col1 string, Updated_Column int)" +
+                "USING delta " +
+                "LOCATION 's3://" + bucketName + "/databricks-compatibility-test-" + tableName + "'" +
+                "TBLPROPERTIES ('delta.enableChangeDataFeed' = true)");
+        try {
+            onTrino().executeQuery("INSERT INTO delta.default." + tableName + " VALUES ('testValue1', 1), ('testValue2', 2), ('testValue3', 3)");
+            onTrino().executeQuery("UPDATE delta.default." + tableName + " SET updated_column = 5 WHERE col1 = 'testValue3'");
+
+            List<Row> expectedRows = ImmutableList.<Row>builder()
+                    .add(row("testValue1", 1, "insert", 1L))
+                    .add(row("testValue2", 2, "insert", 1L))
+                    .add(row("testValue3", 3, "insert", 1L))
+                    .add(row("testValue3", 3, "update_preimage", 2L))
+                    .add(row("testValue3", 5, "update_postimage", 2L))
+                    .build();
+            assertThat(onDelta().executeQuery("SELECT col1, updated_column, _change_type, _commit_version " +
+                    "FROM table_changes('default." + tableName + "', 0)"))
+                    .containsOnly(expectedRows);
+            assertThat(onTrino().executeQuery("SELECT col1, updated_column, _change_type, _commit_version " +
+                            "FROM TABLE(delta.system.table_changes('default', '" + tableName + "'))"))
```

Comment:
nit: the formatting of the statement onTrino seems slightly different than the formatting for onDelta